quoting-service/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: Quoting-Service Helm chart for Kubernetes
 name: quoting-service
 version: 15.8.0
```

Comment:
```suggestion
version: 16.0.0
```

Reply:
Thought we were leaving this for the release ci bumping?